resources/assets/components/pages/VoterRegistrationDrivePage/GroupTypeLink.js

```diff
@@ -55,14 +60,13 @@ const GroupTypeLink = ({ id, name }) => {
         <ErrorBlock error={error || campaignWebsiteError} />
       ) : (
         <div className="mt-3">
-          {loading || campaignWebsiteLoading ? (
-            <PlaceholderText size="medium" />
-          ) : (
+          {loading || campaignWebsiteLoading ? null : (
             <a
               href={groupCampaignUrl}
               target="_blank"
               rel="noopener noreferrer"
               data-testid="voter-registration-drive-page-group-campaign-link"
+              css={groupsLink}
```

Comment:
Non-blocking I'm just curious at this point TBH -- if we use a `style` here, we _should_ be able to override without an `!important` (Emotion also uses classes to assign the styles so I guess it still ends up being lower on the hierarchy then the `.markdown` 🤔 )

```suggestion
              style={{ color: 'white', fontWeight: 700 }}
```

Reply:
fair q, i was able to update the color with emotion but not the font weight, but it does work with a direct style attribute which is still readable and makes sense so i made that change!